app/Http/Controllers/Api/v1/AuthController.php

```diff
@@ -22,18 +22,20 @@ public function index()
     {
         return "auth route";
     }
-    public function login(Request $request)
+
+    /**
+     * Login method for user
+     * @param \App\Http\Requests\Auth\LoginUserRequest $request
+     */
+    public function login(LoginUserRequest $request)
```

Comment:
add return type (check if it returns `JsonResponse`

```suggestion
    public function login(LoginUserRequest $request): JsonResponse
```